go/test/endtoend/vtgate/queries/misc/misc_test.go

```diff
@@ -112,3 +112,15 @@ func TestQueryTimeout(t *testing.T) {
 	_, err = utils.ExecAllowError(t, mcmp.VtConn, "select /*vt+ PLANNER=gen4 QUERY_TIMEOUT_MS=10 */ sleep(0.001) from dual")
 	assert.NoError(t, err)
 }
+
+func TestViews(t *testing.T) {
+	mcmp, closer := start(t)
+	defer closer()
+
```

Comment:
We should also add an end-to-end test that will run VTctld commands to create a view (`CreateViewDDLAction`), run queries using that view, expect the queries to work, and then drop (`DropViewDDLAction`) the view and expect queries to fail.

We can also add another test to test the different error paths defined in `go/vt/topotools/vschema_ddl.go`, where we try to create a view with the same name as an already existing table, where we try to drop a view that does not exist, etc.

I reckon we have unit tests for this in `go/vt/vtgate/executor_vschema_ddl_test.go`, but we don't have E2E tests.

Reply:
There is no vtctld commands for create or drop view.

e2e tests are costly to add, unit tests suffice the purpose here.

Reply:
Right, I meant the DDL queries you've added :)

Reply:
does not apply anymore